chapters/03-current-state.Rmd

```diff
@@ -0,0 +1,27 @@
+# Current state of the art {#currentState}
+
+There are several options available for computing univariate kernel density estimates in Python.
```

Comment:
[5] extent by saying that we will look at them; make a proper introduction. Or motivate. not just a single sentence in the beginning